### PR TITLE
Remove blackfire

### PR DESCRIPTION
**Pull request by jawys**
### Description

This removes 
- the `blackfire*` scripts from Dockerfiles && from repo
- a typo in the `Makefile`
- unnecessary RUN steps due to buildkit features (first set USER, then mkdir+chown+cd using WORKDIR)

### Related issues

See #55

### CODE REVIEW

File: 8.0/Dockerfile [hunk 1]
```diff
@@ -125,20 +125,8 @@ RUN RUNTIME_DEPS="$(scanelf --needed --nobanner --recursive /usr/local \
...
-
-# run the installation script
-RUN set -ex && sh /tmp/blackfire-install.sh && rm -rf /tmp/blackfire*
-
-# make the directories and set permissions
-RUN mkdir -p /app
-
-# set the permissions on the
-RUN chown -R www-data:www-data /app
-
```
Comment (timkelty) [inline 8.0/Dockerfile:139]: @jawys is there a reason you dropped the `chown`?

**Review by timkelty — CHANGES_REQUESTED** [review]

**Comment by timkelty** [comment]
> unnecessary RUN steps due to buildkit features (first set USER, then mkdir+chown+cd using WORKDIR)

this was news to me, thanks!

File: 8.0/Dockerfile (hunk 1 above)
Comment (jawys) [reply 8.0/Dockerfile:139]: see this:
```console
❯ echo "
  FROM php:8.2-fpm-alpine3.18
  # run container as the www-data user
  USER www-data
  # set the working directory for convenience
  WORKDIR /app
  " | docker build - -t test && docker run --rm -it test ls -lad /app
[+] Building 0.6s (6/6) FINISHED                                                          docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                      0.0s
 => => transferring dockerfile: 177B                                                                      0.0s
 => [internal] load .dockerignore                                                                         0.0s
 => => transferring context: 2B                                                                           0.0s
 => [internal] load metadata for docker.io/library/php:8.2-fpm-alpine3.18                                 0.5s
 => [1/2] FROM docker.io/library/php:8.2-fpm-alpine3.18@sha256:adb9e7712d82a8c50fd01b73cd148859c4562bbe8  0.0s
 => CACHED [2/2] WORKDIR /app                                                                             0.0s
 => exporting to image                                                                                    0.0s
 => => exporting layers                                                                                   0.0s
 => => writing image sha256:36352bf42020368e3e118c009f52b17dd7090574a73682f21dbdaf3bae1da351              0.0s
 => => naming to docker.io/library/test                                                                   0.0s
drwxr-xr-x    2 www-data www-data      4096 Aug 29 12:46 /app
```

**Review by timkelty — CHANGES_REQUESTED** [review]

File: 8.0/Dockerfile [hunk 2]
```diff
@@ -125,20 +125,8 @@ RUN RUNTIME_DEPS="$(scanelf --needed --nobanner --recursive /usr/local \
...
-# set the working directory for conveinence
-WORKDIR /app
-
 # run container as the www-data user
 USER www-data
```
[the remark this answers is not in the document]
Comment (jawys) [reply 8.0/Dockerfile:129]: that is why I gave you this: https://github.com/craftcms/docker/pull/83#discussion_r1308768428

the result is the following:
`drwxr-xr-x    2 www-data www-data      4096 Aug 29 12:46 /app`

output of `id`:
```console
❯ docker run --rm -it test id
uid=82(www-data) gid=82(www-data) groups=82(www-data)
```

[the remark this answers is not in the document]
Comment (jawys) [reply 8.0/Dockerfile:129]: Also see https://docs.docker.com/engine/reference/builder/#user

> Note that when specifying a group for the user, the user will have _only_ the specified group membership. Any other configured group memberships will be ignored.

**Comment by jawys** [comment]
@timkelty You marked the conversations as resolved, however I see that you requested changes. What do you want me to do?

**Comment by timkelty** [comment]
@jawys all good – i'll merge when I'm back at a keyboard!

**Review by timkelty — APPROVED** [review]

**Review by timkelty — APPROVED** [review]

**Pull request merged**